prow/kube/client.go

```diff
@@ -492,7 +493,7 @@ func (c *Client) ReplaceProwJob(name string, job ProwJob) (ProwJob, error) {
 	return retJob, err
 }
 
-func (c *Client) CreatePod(p Pod) (Pod, error) {
+func (c *Client) CreatePod(p v1.Pod) (Pod, error) {
```

Comment:
I didn't really need to do these changes until I realized that I should use type aliases for all of our types. Leaving them here since eventually we will want to remove the type aliases.